### ISSUE TITLE
Metric Evaluation

### ISSUE THREAD

**Issue by LWang1016** [NVlabs/Sana#51]
Hi, 
Wonderful work! Currently, when I evaluation the FID with following config on the MJHQ-30K, I got a different number(6.577), which is not match to the paper report(5.76) in Table7. 
Model: Sana-1.6B
image resolution: 1024x1024 
other param: same with the "scripts/bash_run_inference_metric.sh" 
<img width="720" alt="截屏2024-11-27 10 43 02" src="https://github.com/user-attachments/assets/0991205d-dcbb-4846-abb2-167dfc29532c">

Looking forward to your reply.

**Comment by LWang1016** [comment]
Besides, for the DPG evaluation, I saw you need a json file input, so we need convert the dpg prompt/*.txt to one json file, right? share the same structure with MJHQ-30K/meta_data.json?

**Comment by lawrence-cj** [comment]
1. The 5.76 is the result for paper, which is a mid result during training. We fine-tuned the model longer for better image quality. So the released one is a little bit different;
2. Yah, need to convert to a json file, same content, just the format is different.

**Comment by LWang1016** [comment]
@lawrence-cj  thanks for your feedback. have you evaluated the released version? how about the metric number?

**Comment by lawrence-cj** [comment]
We evaluated the GenEval metrics.

<img width="395" alt="image" src="https://github.com/user-attachments/assets/b81fb19d-a03a-432b-97bb-8f1153020ec3">
